packages/@icon-magic/icon-models/src/asset.ts

```diff
@@ -12,8 +12,10 @@ export class Asset {
   name: string;
   contents: Content;
   iconPath: string;
+  private path: string;
   private debug: debugGenerator.IDebugger;
   private unresolvedPath: string;
```

Comment:
Can we get rid of this `unresolvedPath`?

Reply:
It's used in `getPath`: 
```
getPath() {
    return path.resolve(this.iconPath, this.unresolvedPath);
  }
```

Should it be replaced with `this.path`?

Reply:
Did the above.